.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -5,3 +5,5 @@ We thank you for helping improve Crystal. In order to ease the reviewing process
 2. Focus the PR to the referred issue, and restraint from adding unrelated changes/additions. We do welcome another PR if you fixed another issue.
 
 3. If your change is big, consider breaking it into several smaller PRs. In general, the smaller the change, the quicker we can review it.
+
+4. ⚠️ Never force-push the branch your commit lives in. This breaks existing reviews and comments, making the work harder for review. Just merge.
```

Comment:
We have this in `CONTRIBUTING.md` already, maybe we should just copy the wording?
```suggestion
4. ⚠️ Please do not amend previous commits and force push to the PR branch. This makes reviews much harder because reference to previous state is hidden.
```
This is missing an alternative. "Just merge" sounds a bit meager. What about "Instead, just append new commits, including merges of the base branch."

I suppose some general sync between both documents might be a good idea.